.github/workflows/compile_maven.yml

```diff
@@ -15,7 +15,7 @@ jobs:
     strategy:
       matrix:
         os: [ 'ubuntu-latest']
-        java: [ '17' ]
+        java: [ '21' ]
```

Comment:
itt is vannak java 17-ek:
- .github/workflows/maven-deploy.yml
- .github/workflows/manual-branch-RELEASE.yml
- .github/workflows/manual-main-RELEASE.yml